src/components/signIn/Signin.tsx

```diff
@@ -18,6 +18,8 @@ import {
 } from './Signin.styles';
 import { useTranslation } from 'react-i18next';
 import { useSignInMutation } from 'service/signinHttp';
+import { useAppDispatch } from '../../redux/hooks';
+import { saveEmail } from '../../redux/reducers/userSlice';
```

Comment:
absolute path. please check you pr before send to me

Reply:
done